ports/mimxrt10xx/supervisor/port.c

```diff
@@ -306,7 +306,9 @@ void reset_port(void) {
 
     //reset_event_system();
 
+    #if !defined (MIMXRT1062_SERIES)
```

Comment:
Please add a comment here that it is temporary and also create an issue to fix it.